constraint/pkg/apis/templates/v1beta1/zz_generated.conversion.go

```diff
@@ -434,10 +434,8 @@ func autoConvert_v1beta1_Validation_To_templates_Validation(in *Validation, out
 	if in.OpenAPIV3Schema != nil {
 		in, out := &in.OpenAPIV3Schema, &out.OpenAPIV3Schema
 		*out = new(apiextensions.JSONSchemaProps)
-		// TODO: Inefficient conversion - can we improve it?
-		if err := s.Convert(*in, *out, 0); err != nil {
-			return err
-		}
+		// FIXME: Provide conversion function to convert apiextensionsv1beta1.JSONSchemaProps to apiextensions.JSONSchemaProps
+		compileErrorOnMissingConversion()
```

Comment:
this is an issue generating these conversion. not sure why this happens since these conversions exists.

example:
```
func autoConvert_v1beta1_Validation_To_templates_Validation(in *Validation, out *templates.Validation, s conversion.Scope) error {
        if in.OpenAPIV3Schema != nil {
                in, out := &in.OpenAPIV3Schema, &out.OpenAPIV3Schema
                *out = new(apiextensions.JSONSchemaProps)
                if err := v1.Convert_v1_JSONSchemaProps_To_apiextensions_JSONSchemaProps(*in, *out, s); err != nil {
                        return err
                }
        } else {
                out.OpenAPIV3Schema = nil
        }
        return nil
}

func autoConvert_templates_Validation_To_v1beta1_Validation(in *templates.Validation, out *Validation, s conversion.Scope) error {
        if in.OpenAPIV3Schema != nil {
                in, out := &in.OpenAPIV3Schema, &out.OpenAPIV3Schema
                *out = new(v1.JSONSchemaProps)
                if err := v1.Convert_apiextensions_JSONSchemaProps_To_v1_JSONSchemaProps(*in, *out, s); err != nil {
                        return err
                }
        } else {
                out.OpenAPIV3Schema = nil
        }
        return nil
}
```

Reply:
There's already a k8s bug for this (kubernetes/kubernetes#98380)

Reply:
False alarm!  I just needed to make that package available to `conversion-gen`.  Thanks for posting that example @sozercan, that was the clue I needed.

Reply:
Issue says it's for `types containing an array of pointers to custom objects.` Does this apply to our types?

Reply:
Sorry, it isn't completely resolved.  I've included notes in `Makefile`.

But, I've got us frankensteined together enough to move forward.